### PR TITLE
Copy the map_v4v6_address function from glibc

### PR DESCRIPTION
**Pull request by floppym**
Fixes: https://github.com/thkukuk/libnss_nis/issues/1

### CODE REVIEW

File: src/Makefile.am [hunk 1]
```diff
@@ -27,4 +27,5 @@ libnss_nis_la_LIBADD = @LIBNSL_LIBS@
 libnss_nis_la_SOURCES = nis-alias.c nis-ethers.c nis-grp.c nis-hosts.c \
 			nis-initgroups.c nis-netgrp.c nis-network.c  \
 			nis-proto.c nis-publickey.c nis-pwd.c nis-rpc.c \
-			nis-service.c nis-spwd.c nss-nis.c nss-default.c
+			nis-service.c nis-spwd.c nss-nis.c nss-default.c \
+			map_v4v6_address.h
```
Comment (thkukuk) [inline src/Makefile.am:31]: This change is wrong, map_v4v6_address.h is no C file which should be compiled and linked into the library but a header which should not be installed.

**Comment by thkukuk** [comment]
I commited a correct version to git.

**Comment by thkukuk** [comment]
I commited a correct version to git.

File: src/Makefile.am (hunk 1 above)
Comment (floppym) [reply src/Makefile.am:31]: automake seems to be smart enough to not compile files ending with ".h".

Anyway, your way seems fine.

**Pull request closed without merging**